jax/_src/lax/lax.py

```diff
@@ -1757,10 +1761,17 @@ def _round_lower(ctx, x, *, rounding_method):
 
 exp_p = standard_unop(_float | _complex, 'exp')
 ad.defjvp2(exp_p, lambda g, ans, x: mul(g, ans))
-# For exp_p it is more efficient to use the reconstructed output for the vjp
```

Comment:
Not applicable any more?

Reply:
Well, it was both inaccurate and pointless: inaccurate because it talks about a VJP rule when no VJP rule is defined (it's a JVP rule) and pointless because that's what `ad.defjvp2` always does (i.e. passes in `ans` so it can be used).

Reply:
Actually I think that comment pertained to a now-deleted rule for invertible autodiff (in #3232).